src/vm/disassembler.cpp

```diff
@@ -139,6 +171,7 @@ void Disassembler::StaticInitialize()
 
     CLRFreeLibrary(libraryHandle);
     _ASSERTE(!IsAvailable());
+    
```

Comment:
I know this is not your change for CLRFreeLibrary. But it appears we run this even in the case libraryHandle = nullptr, and I wonder what the consequence is. It's better to skip this if we didn't create libraryHandel at all.